### ISSUE TITLE
Unhandled exception when closing Gdevelop before a preview window

### ISSUE THREAD

**Issue by edrejon89** [4ian/GDevelop#3997]
### Description 
When Gdevelop is closed before closing an open preview game window an unhandled exception error is thrown.

### To Reproduce
1. Open Gdevelop
2. Open game preview
3. Close Gdevelop
4. Close the preview window

![image](https://user-images.githubusercontent.com/35882707/172193780-eaca2d86-e770-44ff-a03e-6059e5e43782.png)

![zTqexdB90c](https://user-images.githubusercontent.com/35882707/172197664-44c71f43-61a9-498b-b87c-09fcfd190464.gif)

## Other details
* OS: Windows 10 Pro
* Platform: Desktop App

**Comment by Oxey405** [comment]
Update : this bug has appently already been fixed so if it still occurs on your computer, try updating or re-installing GDevelop.

**Comment by 4ian** [comment]
I don't think it was fixed, this is I think appearing since 1 or 2 versions, as the preview window was moved from the renderer process to the electron main process :)

**Comment by ClementPasteau** [comment]
should be fixed now
